src/scores/dates.py

```diff
@@ -2,15 +2,15 @@
 from pandas import DataFrame, Timestamp
 
 
-def main(df: DataFrame) -> DataFrame:
+def main(checks: DataFrame) -> DataFrame:
     """Draft function for scoring date values within dataset.
 
     Args:
-        df: checks DataFrame.
+        checks: checks DataFrame.
 
     Returns:
         Checks DataFrame with additional columns for scoring.
     """
-    df_score = df[["iso3", "level"]].copy()
-    df_score["dates"] = df["update_1"] > Timestamp.now() - relativedelta(years=3)
-    return df_score
+    scores = checks[["iso3", "level"]].copy()
+    scores["dates"] = checks["update_1"] > Timestamp.now() - relativedelta(years=1)
```

Comment:
Where did the 3 / 1 year magic numbers come from?

Reply:
The old 3 years was a magic number I made up as a placeholder value. There is actually an annual COD review process where datasets should be validated on a 12 month cycle. (source: https://cod.unocha.org/)

<img width="386" alt="Screenshot 2024-10-06 at 15 47 22" src="https://github.com/user-attachments/assets/7c4e2965-b898-4f5d-bb1d-ce4ff85fc55f">

Reply:
Thanks for explaining - I think that's worth adding into a comment, I don't think the next dev in 6 months will be able to guess this 🙂